### PR TITLE
Add support for absolute paths when setting server dbd

### PR DESCRIPTION
**Pull request by joshuafcole**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -33,7 +33,13 @@ module.exports = function init(opts) {
   var server = new Hapi.Server('0.0.0.0', opts.port, serverOpts);
 
   server.settings.app = opts;
-  server.settings.app.dbd = path.join(opts.cwd, (opts.dbd || 'run'));
+  var dbd = opts.dbd || 'run';
+  // This is awful, but normalize doesn't remove trailing slashes, and we can't compare against dbd directly in case it isn't normalized (e.g. /foo/../)
+  if(path.resolve(dbd) === path.normalize(dbd).replace( RegExp(path.sep+'$'), '' )) {
```
Comment (diffsky) [inline server.js]: Can we just check if opts.dbd is absolute?
```
if (path.isAbsolute(opts.dbd)) {
  server.settings.app.dbd = dbd;
} else {
  server.settings.app.dbd = path.join(opts.cwd, (opts.dbd || 'run'));
}
```

Comment (joshuafcole) [inline server.js]: I had no idea that existed, will update momentarily.

**Comment by diffsky** [comment]
@joshuafcole I've added support for this in https://github.com/creativelive/spook/commit/3b31909559543c22ecbe5236484d69e9e197efe6

**Comment by joshuafcole** [comment]
Thanks Matt!

**Pull request closed without merging**